README.md

```diff
@@ -2,6 +2,25 @@
 
 Utilities for testing Drupal!
 
+## ErrorMessageDetectionTrait
+
+A trait to detect error messages in `drupalGet` responses and display the error message when the test fails.
+
+### Usage
+
+Once your trait is added to your test base class, you can check detect errors by overriding the `drupalGet` function.
```

Comment:
Would be nice if this also covered form submissions and other link clicks. One way to achieve this would be to override the htmlOutput function and detect the error then. The html output contains other messages before the page content so we can't use str_starts_with anymore, and may need to strip some other cruft. Maybe it could check `str_contains` in the `$message` and then call `$this->getSession()->getPage()->getContent()` again to get the error to display.